tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -267,6 +267,29 @@ def _run(self):
                                     rebal_files=volume.rebal_files,
                                     rebal_data=volume.rebal_data
                                 ).save()
+                    args = ["gstatus", "-s"]
+                    p = subprocess.Popen(
```

Comment:
Use https://github.com/Tendrl/commons/blob/develop/tendrl/commons/utils/cmd_utils.py instead of subprocess